oracle/README.md

```diff
@@ -207,6 +205,11 @@ If you are not using JDBC, verify that the Datadog Agent is able to connect to y
 sqlplus <USER>/<PASSWORD>@(DESCRIPTION=(ADDRESS_LIST=(ADDRESS=(PROTOCOL=TCPS)(HOST=<HOST>)(PORT=<PORT>))(SERVICE_NAME=<SERVICE_NAME>)))
 ```
 
+When using the [Oracle Instant Client][16] connection, move three files to the `network/admin` directory of the client libraries used by your application:
+  * `tnsnames.ora`: Maps net service names used for application connection strings to your database services.
+  * `sqlnet.ora`: Configures Oracle Network settings.
+  * `cwallet.sso`: Enables SSL or TLS connections in thick mode. Keep this file secure.
```

Comment:
I saw that in [this comment](https://github.com/DataDog/integrations-core/pull/14166#discussion_r1145129203), we remove reference to the python-oracledb library thick mode, although I think if we mention "thick mode" in the README, we should still mention python-oracledb since that term is specific to the library. Another option is we don't mention thick mode and be library agnostic in the README. WDYT @hithwen?

Reply:
We need to stick to one way or the other, either we say "native client", "instant client" or "thick mode" but let's use the same terminology everywhere